lib/moon/components/label.ex

```diff
@@ -3,7 +3,7 @@ defmodule Moon.Components.Label do
 
   use Moon.StatelessComponent
 
-  prop(color, :css_class, default: "text-goten")
```

Comment:
sure this `gotten` should be replaced also?

Reply:
reverted